src/plugins/management/public/types.ts

```diff
@@ -90,6 +90,7 @@ export interface CreateManagementItemArgs {
   title: string;
   tip?: string;
   order?: number;
+  showExperimentalBadge?: boolean;
```

Comment:
we may consider to support other badge types to make this more generic. e.g. let outer component to optionally pass a badge type.
Not a blocker for this change though.